fb-setup-discovery.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+load os_helper
+load foreman_helper
+
+@test "install discovery plugin" {
+  yum -y install tfm-rubygem-foreman_discovery tfm-rubygem-hammer_cli_foreman_discovery
+}
+
+@test "create discover-host script" {
```

Comment:
> To me that sounds like we provide insufficient debug info during the normal process. If we'd test this under Jenkins that sounds like something we'd want.

Frankly, I use BATS to have "everyday" instances I play with. Having a
helper ready is quite handy.

-- 
Later,
 Lukas #lzap Zapletal